picture-game/src/Content/Question.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {useEffect, useState} from 'react';
+import styled from 'styled-components'
+import gameData from '../GameData.js';
+
+const QuizPicture = styled.img`
+    width: 15rem;
+    height: 20rem;
+`
+const Answers = styled.div`
+    display: flex;
+`
+const Answer = styled.button`
+    color: white;
+    background: purple;
+    border-radius: 5px;
+    width: 6rem;
+    height: 2rem;
+    font-size: 18px;
+`
+
+export default function Question() {
+    const [currentGameData, setCurrentGameData] = useState([...gameData][0]);
+    const chooseAnswer = () => {}
+    useEffect(() => {
+        setCurrentGameData([...gameData][0])
+    })
```

Comment:
```suggestion
useEffect(() => {
        setCurrentGameData([...gameData][0])
    },[])
```
[useEffect 두번째 인자](https://wonyong-jang.github.io/react-redux/2021/05/21/React-useState-useEffect.html)를 설정해주는게 불필요한 리렌더링을 막을수 있는데에!! 혹시 뺀 이유가 잇을까?1

Reply:
뺀건 아니고 코드 작성하다가 내가 누락한것 같아...!